scripts/main.ts

```diff
@@ -403,13 +411,17 @@ export async function initializeProposal(
       daoTreasury
     );
 
-  const cuIx = ComputeBudgetProgram.setComputeUnitPrice({
+  const cuPriceIx = ComputeBudgetProgram.setComputeUnitPrice({
     microLamports: 100,
   });
+  const cuLimitIx = ComputeBudgetProgram.setComputeUnitLimit({
+    units: 150_000
+  });
 
   let tx1 = new Transaction();
   tx1.add(...passMarketInstructions);
-  tx1.add(cuIx);
+  tx1.add(cuPriceIx);
```

Comment:
This looks like the compute budget program instructions get called after the pass market instructions? Does it matter what order the instructions get called in. Only asking because in an early function they're added as preInstructions.